src/rgw/rgw_putobj_processor.h

```diff
@@ -157,6 +163,7 @@ class AtomicObjectProcessor : public ManifestObjectProcessor {
   const std::optional<uint64_t> olh_epoch;
   const std::string unique_tag;
   bufferlist first_chunk; // written with the head in complete()
+  const DoutPrefixProvider *dpp;
```

Comment:
AtomicObjectProcessor already inherits a dpp from ManifestObjectProcessor, so doesn't need to store an extra pointer here. same for the Multipart and Append processors